main.js

```diff
@@ -0,0 +1,212 @@
+const difficulties = document.querySelectorAll("input[name='difficulty']");
+const timer = document.getElementById('timer');
+const refresh = document.getElementById('refresh');
+const replay = document.getElementById('replay');
+const modal = document.querySelector('.modal');
+let moveCount = 0;
+let selectedCards = [];
+let cardMatch, NumberOfCards, difficulty, sec, moves, correctMoves, setTimer;
+
+//-----SHUFFLE FUNCTION
+function shuffleCards(list) {
+    for (let i = 0; i < list.length; i += 1) {
+        let j = Math.floor(Math.random() * list.length);
+
+        let temp = list[i];
+        list[i] = list[j];
+        list[j] = temp;
+    }
+}
+
+
+function populate(num) {
+	cardMatch = [];
+	moveCount = 0;
+	board.innerHTML = '';
+	//shuffle (cardList) and take half the number of cards based on difficulty to get a random icons every game
+	shuffleCards(cardList);
+	let cards = cardList.slice(0, num/2);
+	//duplicate the array values to make pairs and shuffle the new array to get random order of cards
+	cards = cards.concat(cards);
+	shuffleCards(cards);
+	//populate HTML
+	const fragment = document.createDocumentFragment();
+	for (let x = 0; x < num; x += 1) {
+		const cardContainer = document.createElement('div');
+		cardContainer.classList.add('card-container', difficulty);
+		const front = document.createElement('div');
+		const back = document.createElement('div');
+		front.classList.add('card', 'front');
+		back.classList.add('card', 'back');
+		const icon = document.createElement('span');
+		icon.classList.add('icon', 'material-symbols-outlined');
+		icon.setAttribute("name", cards[x]);
+		icon.innerText = cards[x];
+		back.appendChild(icon);
+		cardContainer.appendChild(front);
+		cardContainer.appendChild(back);
+		fragment.appendChild(cardContainer);
+	}
+	board.appendChild(fragment);
+	console.log(cards);
+}
+
+
+//-----CHOOSE DIFFICULTY
+function chooseDifficulty(){
+	[].forEach.call(difficulties, function(input){
+		input.nextElementSibling.classList.remove('checked');
+		console.log(input.nextElementSibling)
+		if (input.value === 'easy' && input.checked === true) {
+			NumberOfCards = 12;
+			difficulty = 'easy';
+			input.nextElementSibling.classList.add('checked');
+			board.classList.remove('normal-game', 'hard-game');
+			board.classList.add('easy-game');
+		} else if (input.value === 'normal' && input.checked === true) {
+			NumberOfCards = 24;
+			difficulty = 'normal';
+			input.nextElementSibling.classList.add('checked');
+			board.classList.remove('easy-game', 'hard-game');
+			board.classList.add('normal-game');
+		} else if (input.value === 'hard' && input.checked === true) {
+			NumberOfCards = 36;
+			difficulty = 'hard';
+			input.nextElementSibling.classList.add('checked');
+			board.classList.remove('easy-game', 'normal-game');
+			board.classList.add('hard-game');
+		}
+	});
+}
+
+function stopwatch(){
+	sec += 1;
+	let minutes = Math.floor(sec/60);
+	let seconds = sec % 60;
```

Comment:
Aceste variabile nu sunt schimbate. Le poți defini cu `const`

```suggestion
	const minutes = Math.floor(sec/60);
	const seconds = sec % 60;
```

Reply:
Am schimbat